src/TabNav.tsx

```diff
@@ -2,19 +2,34 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import styled from 'styled-components'
-import {COMMON, get} from './constants'
+import {COMMON, SystemCommonProps, SystemTypographyProps, get} from './constants'
+import {ComponentProps} from './utils/types'
 import theme from './theme'
-import sx from './sx'
+import sx, {SxProp} from './sx'
+import * as History from 'history'
 
 const ITEM_CLASS = 'TabNav-item'
 const SELECTED_CLASS = 'selected'
 
-function TabNavBase({actions, className, align, children, full, theme, ...rest}) {
+const TabNavParent = styled.nav`
+  ${COMMON}
+  ${sx}
+`
```

Comment:
This felt like a weird thing to do - I was trying to parallel some patterns in `src/UnderlineNav.tsx` and I could not get the props for L19 unless I had an existing Parent element here.

This also seems to be producing some console errors in the tests (though they still pass):
![image](https://user-images.githubusercontent.com/16739070/107295494-17cc2d80-6a35-11eb-9121-63deab163f3d.png)

Not sure if the tests needs to be adjusted or if trying to pull out a parent like this is just a bad idea?